server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -23,43 +21,31 @@
   protected void doGet(HttpServletRequest req, HttpServletResponse res)
       throws ServletException, IOException {
 
-    res.getWriter().write("<h1>Comment Analysis</h1>\n");
-
-    String videoName = "seun omonije";
+    // Video name from the front-end
+    String videoName = "sharks 101";
 
+    // Get response from Youtube API
     String videoIdJson = new YoutubeRequest("search", videoName).getResult();
     String videoId = getVideoId(videoIdJson);
     String commentsJson = new YoutubeRequest("commentThreads", "snippet", videoId).getResult();
 
-    res.getWriter().write(commentsJson);
-
-    res.getWriter().write("<h3>Comments</h3>");
-    String comment = "I love you.";
-    res.getWriter().println(comment);
+    // Convert response into a large string to be passed through for analysis
+    ArrayList<String> commentArray = retrieveComments(commentsJson);
+    String cumulativeComments = convertToString(commentArray);
 
-    res.getWriter().write("<h3>Perspective</h3>");
-    HashMap<String, String> perspectiveMap = analyzeWithPerspective(comment);
-    // Print on the screen (won't make it in final cut)
-    Set set = perspectiveMap.entrySet();
-    Iterator mapIterator = set.iterator();
-    while (mapIterator.hasNext()) {
-      Map.Entry entry = (Map.Entry) mapIterator.next();
-      res.getWriter().println(entry.getKey() + ": " + entry.getValue());
-    }
+    HashMap<String, String> perspectiveMap = analyzeWithPerspective(cumulativeComments);
+    MagnitudeAndScore nlpObject = analyzeWithNLP(cumulativeComments);
```

Comment:
nit: name this variable something more descriptive like "sentiment" or "commentsSentiment"